app/assets/stylesheets/steps.scss

```diff
@@ -25,3 +25,28 @@ pre.markdown-example {
 .table-stepnav-rules .checkbox label {
   margin-top: 0;
 }
+
+.broken-links-accordion__icon-text {
+  margin-right: 3px;
+}
+
+.broken-links-accordion__icon,
+.broken-links-accordion__icon-text {
+  float: right;
+}
+
+.broken-links-accordion__icon--show.glyphicon,
```

Comment:
This class combination looks unnecessarily complicated. Can I suggest wrapping both the icon and the text in a single element, then showing/hiding that single element using the technique you're using?